src/www/css/app.css

```diff
@@ -1532,8 +1584,12 @@ video {
     display: inline-block;
   }
 
-  .md\:flex-row {
```

Comment:
Can I just check that we deffo want to get rid of `flex-row`?

Reply:
Yep, haven’t seen any reports of a bug from nightly users until now. :)